### PR TITLE
feat: filter by service and city

### PR DESCRIPTION
**Pull request by amanuela97**
As a user, I would like to filter services based on the city and the service type. see https://trello.com/c/TnFo7mCd/310-filter-by-city & https://trello.com/c/5To9j8Z2/311-filter-by-service

## Acceptance Criteria
- [x] route `/services` displays all services
- [x] route `/services/:location?/ `displays services that match the location name
- [x] route `/services/:location?/:service?/` displays services that match the service type 
- [x] spinner is shown when data is loading

## Demo
https://deploy-preview-76--wehelpvip.netlify.app/services/

### CODE REVIEW

File: pages/services/index.vue [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+export default {
+  name: 'Services',
+  data: () => ({
+    services: [],
+  }),
+  async mounted() {
+    const filter = this.$route.params
+    this.services = await fetchServices(null, filter)
+    console.log(this.services)
```
Comment (razbakov) [inline pages/services/index.vue:19]: remove debug info

**Review by razbakov — CHANGES_REQUESTED** [review]
1) include direct link to test in your PR, i.e. https://deploy-preview-76--wehelpvip.netlify.app/
2) https://deploy-preview-76--wehelpvip.netlify.app/services/Munich - no results
3) I would rather do filter client-side, we don't have that many services yet, and it will work faster with less API requests

File: utils/airtable.js [hunk 2]
```diff
@@ -10,7 +10,20 @@ export const fetchServices = async (offset = null) => {
...
+      const value =
+        val
+          .toLowerCase()
+          .charAt(0)
+          .toUpperCase() + val.slice(1)
+      const array = ['Services'].includes(name)
+      if (val) args.push({ name, value, isArray: array })
+    }
+    const param = args.length > 0 ? filterByFieldName(args) : ''
+    console.log(param)
```
Comment (razbakov) [inline utils/airtable.js:25]: remove debug info

File: utils/airtable.js [hunk 3]
```diff
@@ -10,7 +10,20 @@ export const fetchServices = async (offset = null) => {
...
-    const url = `https://api.airtable.com/v0/${config.BASE_ID}/Services?view=Grid%20view&${offSet}`
+    const args = []
+    for (const [key, val] of Object.entries(filter)) {
+      const name = key.charAt(0).toUpperCase() + key.slice(1)
+      const value =
+        val
+          .toLowerCase()
+          .charAt(0)
+          .toUpperCase() + val.slice(1)
+      const array = ['Services'].includes(name)
```
Comment (razbakov) [inline utils/airtable.js:21]: that's strange

**Review by razbakov — CHANGES_REQUESTED** [review]
please change card border to gray like in https://wehelp.live/guests ?

File: nuxt.config.js [hunk 4]
```diff
@@ -1,4 +1,7 @@
+import { resolve } from 'path'
```
Comment (razbakov) [inline nuxt.config.js:1]: see code example in https://v3.nuxtjs.org/examples/advanced/module-extend-pages/

```js
import { resolve } from 'pathe'
```

**Review by razbakov — CHANGES_REQUESTED** [review]

**Comment by Simran191997** [comment]
route /services displays all services
 route /services/:location?/ displays services that match the location name
 spinner is shown when data is loading

Above mentioned acceptance criteria is working fine  in android device ,ipad and laptop.

/services/:location?/:service?/ displays services that match the service type 

only the above one is not working fine.

**Pull request merged**